modules/userId/eids.js

```diff
@@ -277,6 +277,11 @@ export const USER_IDS_CONFIG = {
     atype: 3
   },
 
+  'imppid': {
+    source: 'ppid.intimatemerger.com',
+    atype: 1
```

Comment:
you likely want unit tests for this too in the eids spec file

Reply:
add test